miqa/core/rest/project.py

```diff
@@ -56,13 +56,23 @@ def get_default_email_recipients(self, obj):
 class ProjectTaskOverviewSerializer(serializers.ModelSerializer):
     class Meta:
         model = Project
-        fields = ['total_experiments', 'total_scans', 'my_project_role', 'scan_states']
+        fields = [
+            'total_experiments',
+            'total_scans',
+            'my_project_role',
+            'scan_states',
+            'project_id',
+        ]
 
+    project_id = serializers.SerializerMethodField('get_project_id')
```

Comment:
```suggestion
    project_id = serializers.UUIDField(source='id')
```

Then you can get rid of the method below.